src/utils/index.ts

```diff
@@ -42,6 +42,16 @@ export const wrapErrorHandler = (fn: (rq: Request, rs: Response) => Promise<void
   };
 };
 
+/**
+ * Returns second to last value in a relatedUrl
+ * to be used as a key for thumbnail assets link.
+ * Defaults to string 'key' in the case of an unexpected URL format.
+ */
+export const extractAssetMapKey = (relatedUrl: string) => {
+  const urlArray = relatedUrl.split(".");
+  return urlArray[urlArray.length - 2] ? urlArray[urlArray.length - 2] : "key";
```

Comment:
add the test for the case where it has to default to "key"

Reply:
Updated tests to verify this code is working as intended